src/mlpack/tests/augmented_rnns_tasks_test.cpp

```diff
@@ -0,0 +1,291 @@
+/**
+ * @file augmented_rnns_tasks.cpp
+ * @author Konstantin Sidorov
+ *
+ * Tests the rtasks for augmented network models.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <vector>
+#include <algorithm>
+#include <utility>
+
+#include <mlpack/core.hpp>
+
+#include <mlpack/core/data/binarize.hpp>
+
+#include <mlpack/methods/ann/augmented/tasks/copy.hpp>
+#include <mlpack/methods/ann/augmented/tasks/sort.hpp>
+#include <mlpack/methods/ann/augmented/tasks/add.hpp>
+#include <mlpack/methods/ann/augmented/tasks/score.hpp>
+
+#include <mlpack/core/optimizers/adam/adam.hpp>
```

Comment:
Looks like we can remove a bunch of the ann header here, since we just test the tasks without using a network?